tests/testthat/test_functions.R

```diff
@@ -0,0 +1,13 @@
+context("Tests of package functions")
+
+mock_data <- data.table::data.table(va = c(1, 5, 4), vb = c(3, 7, 6), vc = c("one", "two", "three"))
```

Comment:
I'd also add some tests that check if functions return correct values, for example:
```
expect_equal(sum_columns(mock_data)[[1]], 10)
```

Please add also factor column to mock data, in order to test output for factors.

Is there any reason why did you use data.table instead of data.frame?